ansible/templates/usb/scripts/computenode.sh.j2

```diff
@@ -122,64 +131,109 @@ function install_archives() {
 }
 
 function __zk() {
-	zkcli="${USB_COPY}/scripts/zookeepercli"
-	args="${@}"
+	cmd="${@}"
 
-	"${zkcli}" -servers "${CFGDB_IP}:2181" -auth_usr "esdc" -auth_pwd "${ESDC_INSTALL_PASSWORD}" ${args}
+	"${QUERY_CFGDB}" ${cmd}
 }
 
 function _zk() {
-	if ! __zk -c "${@}" 2> /dev/null; then
+	if ! __zk "${@}" 2> /dev/null; then
 		fatal "Error: cfgdb command \"${@}\" failed"
 	fi
 }
 
+function _zk_noerror() {
+	if ! __zk "${@}" 2> /dev/null; then
+		# return success anyway
+		return 0
+	fi
+}
+
+# omit SSL certificate verification
+function _zk_noverify() {
+	_zk_noerror -k "${@}"
+}
+
+test_cfgdb_connect() {
+	if [[ -z "$(_zk get /esdc)" ]]; then
```

Comment:
Querying the top-level /esdc is forbidden in our new ZK rest service, e.g. query /esdc/settings.